viewing_party/party.py

```diff
@@ -0,0 +1,287 @@
+def create_movie(movie_title, genre, rating):
+    """
+    input: 3 strings which provide details about a movie
+    output: a dictionary which stores details about the movie
+    """
+
+    # Create a dictionary where the keys are
+    # title, genre, and rating of
+    # the new dictionary - grab these values
+    # from the arguments passed as parameters
+
+    # if every argument that is passed is truthy
+    # then run the code below, otherwise:
+    # run different code
+
+    if movie_title and genre and rating:
+        movie = {}
+        movie["title"] = movie_title
+        movie["genre"] = genre
+        movie["rating"] = rating
+        return movie
+    else:
+        return None
```

Comment:
Consider first testing for the edge case, and then moving the main functionality of your function outside the `if/else`. This programming pattern is called a guard clause and can among other things enhance the readability of your code.

```suggestion
    if not movie_title and not genre and not rating:
        return None
        
    movie = {}
    movie["title"] = movie_title
    movie["genre"] = genre
    movie["rating"] = rating
```